.github/workflows/k3s-versions.yaml

```diff
@@ -0,0 +1,32 @@
+name: Update k3s-versions.json
+on:
+  schedule:
+  - cron: '43 8 * * *'
+  workflow_dispatch: {}
+
+permissions:
+  contents: write
+  pull-requests: write
+
+jobs:
+  check-for-token:
+    outputs:
+      has-token: ${{ steps.calc.outputs.HAS_SECRET }}
+    runs-on: ubuntu-latest
+    steps:
+    - id: calc
+      run: echo "HAS_SECRET=${HAS_SECRET}" >> "${GITHUB_OUTPUT}"
+      env:
+        HAS_SECRET: ${{ secrets.RUN_WORKFLOW_FROM_WORKFLOW != '' }}
```

Comment:
Please use https://github.com/rancher-sandbox/rancher-desktop/blob/main/.github/actions/get-token/action.yaml instead.  The upstream repo no longer has a `RUN_WORKFLOW_FROM_WORKFLOW` secret (you need to get it dynamically from vault).

Reply:
How is this possible? This is a direct copy from `rddepman.yaml`:

```diff
--- rddepman.yaml	2024-11-01 18:57:15.000000000 -0700
+++ k3s-versions.yaml	2024-12-11 14:04:28.000000000 -0800
@@ -1,7 +1,7 @@
-name: Update external dependencies
+name: Update k3s-versions.json
 on:
   schedule:
-    - cron: '23 8 * * *'
+  - cron: '43 8 * * *'
   workflow_dispatch: {}

 permissions:
@@ -24,13 +24,9 @@
     if: needs.check-for-token.outputs.has-token == 'true'
     runs-on: ubuntu-latest
     steps:
-
       - uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
         with:
           fetch-depth: 0
-
-      - uses: ./.github/actions/yarn-install
-
-      - run: yarn rddepman
+    - run: ./scripts/k3s-versions.sh
         env:
           GITHUB_TOKEN: ${{ secrets.RUN_WORKFLOW_FROM_WORKFLOW }}
```

Reply:
Huh. Maybe I misremembered why we needed to use the vault secrets? That was around when we needed to manually retrigger CI (close and re-open PRs).